.expeditor/scripts/bk_win_functional.ps1

```diff
@@ -20,10 +45,10 @@ Write-Output "--- configure winrm"
 winrm quickconfig -q
 
 Write-Output "--- bundle install"
-bundle config set --local without 'omnibus_package'
+bundle config set --local without "omnibus_package"
```

Comment:
why we had to do this?

Reply:
I got used to getting linting errors about single vs double quotes and updated this while I was checking things out.